etabotsite/etabotsite/settings.py

```diff
@@ -179,6 +179,12 @@
 
 USE_TZ = True
 
+SYS_DOMAIN = ''
```

Comment:
for SYS_DOMAIN please use boolean LOCAL_MODE to set to the right value automatically.